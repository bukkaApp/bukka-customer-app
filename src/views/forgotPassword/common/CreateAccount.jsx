import React from 'react';

import PropTypes from 'prop-types';
import Button from 'Components/button/Button';
import './authfooter.scss';

const signUpTextOption = 'New to Bukka?';

const CreateAccount = ({
  push
}) => {
  const navigateToAuth = ({ target: { id } }) => {
    push(id);
  };

  return (
    <div className="form-options padding">
      <p>{signUpTextOption}</p>
      <Button
        id="/signup"
        type="button"
        handleClick={navigateToAuth}
        classNames="btn-link auth-footer-btn"
        text="SIGN UP"
      />
    </div>
  );
};

export default CreateAccount;

CreateAccount.defaultProps = {
  push: () => {},
};

CreateAccount.propTypes = {
  push: PropTypes.func,
};

