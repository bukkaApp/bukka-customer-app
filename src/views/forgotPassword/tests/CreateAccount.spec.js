import React from 'react';
import CreateAccount from '../common/CreateAccount';


describe('Create Account component', () => {
  const props = {
    push: jest.fn(),
  };
  const wrapper = mount(<CreateAccount {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
