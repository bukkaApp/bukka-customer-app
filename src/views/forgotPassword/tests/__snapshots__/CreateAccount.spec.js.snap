// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Account component renders properly 1`] = `ReactWrapper {}`;
